golang/pkg/ceresprompb/prometheus.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.20.1
+// 	protoc        v3.21.12
```

Comment:
Check you local `protoc` version, keep same as before.

Reply:
ok，done